src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
+import react from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] it would be better if `MIT` link on you `README` file is working and you should add a proper MIT license so that your project will be pro